cvat/apps/documentation/backup_hld.md

```diff
@@ -0,0 +1,57 @@
+## Task and Project Import/Export functionality
+
+This document describes the high-level design for implementing import / export implementation for tasks and projects.
+
+API endpoints:
+
+- Import task
+
+  - endpoint: `/api/v1/tasks?action=import​`
+  - method: `POST`
+  - Content-Type: `multipart/form-data​`
+  - returns: json
+
+- Export task
+
+  - endpoint: `/api/v1/tasks/{id}?action=export​`
+  - method: `GET`
+  - returns: zip archive
+
+  The zip archive has the following structure:
+
+  ```
+   .
+   ├── data
+   │   ├── compressed
```

Comment:
Does it really have compressed and original directories?

Reply:
Yes, if the task was created with `use_cache=false` option. In some cases the task does not contain raw data and we cannot recreate it from raw. Anyway, I think that recreating the task from the raw data could lead to potential problems (e.g. different versions of ffmpeg).